pdns/gss_context.hh

```diff
@@ -0,0 +1,151 @@
+#ifndef GSS_CONTEXT_HH
```

Comment:
I know we haven't been doing this too often in the past, but is it possible to have a small docblock for these functions that describes their purpose?